WebHostLib/static/styles/hostRoom.css

```diff
@@ -55,4 +55,7 @@
     border: 1px solid #2a6c2f;
     border-radius: 6px;
     color: #000000;
+	overflow-y: auto;
+	overflow-x: hidden;
```

Comment:
It is undesirable to hide the horizontal overflow in case a long string of text is displayed. I also recommend reducing the y-height from 600 down to 400. It fits nicely on a 1080p monitor in cases where two only two players are involved in a game.

This otherwise seems fine.

Reply:
The only reason i am hiding the horizontal is because long messages would be just carried to the next line, unless you have a long string of one word which shouldnt happen